hadoop-hdds/docs/content/feature/SCM-HA.md

```diff
@@ -234,7 +228,7 @@ bin/ozone debug ldb --db=/tmp/metadata/scm.db scan --column-family=containers
 
 ## Migrating from existing SCM
 
-SCM HA can be turned on on any Ozone cluster. First enable Ratis (`ozone.scm.ratis.enable`) and configure only one node for the Ratis ring (`ozone.scm.nodes.serviceId` should have one element).
+SCM HA is turned on for any Ozone cluster after upgrade to 2.0.0 or later. First configure only one node for the Ratis ring (`ozone.scm.nodes.serviceId` should have one element).
 
 Start the cluster and test if it works well.
```

Comment:
```suggestion
Add additional SCM nodes and extend the cluster configuration to reflect the newly added nodes. 
Bootstrap the newly added SCM nodes with `scm --bootstrap` command and start the SCM service.
Note: Make sure that the `ozone.scm.primordial.node.id` property is pointed to the existing SCM before you run the `bootstrap` command on the newly added SCM nodes.
```

Reply:
It needs to be more prescriptive than this. For now I am fine with this, but I'd like to update it with more step-by-step instructions and examples if I get a chance to do this myself.